index.js

```diff
@@ -93,8 +93,11 @@ ProcessReporter.prototype.bootstrap = function bootstrap() {
     self.memoryTimer = self.timers.setTimeout(onMemory, self.memoryInterval);
     self.lagTimer = self.timers.setTimeout(onLag, self.lagInterval);
 
-    self.gcStats = new _gcstats();
-    self.gcStats.on('stats', self._onStatsListener);
+    _gcstats.afterGC(onStats);
```

Comment:
what if i just move that method inside the if statement above so it is only called once ever?